lib/chef/platform/rebooter.rb

```diff
@@ -35,7 +35,9 @@ def reboot!(node)
 
           cmd = if Chef::Platform.windows?
                   # should this do /f as well? do we then need a minimum delay to let apps quit?
-                  "shutdown /r /t #{reboot_info[:delay_mins] * 60} /c \"#{reboot_info[:reason]}\""
+                  # Use explicit path to shutdown.exe, to protect against https://github.com/chef/chef/issues/5594
+                  windows_shutdown_path = 'C:/Windows/System32/shutdown.exe'
```

Comment:
I don't think so (re: `Which`), the original bug was because there was something else higher up on the path that happens to be called `shutdown`. 👍 that this should be using `ENV['SystemDrive']` or whatever it is now though.